### PR TITLE
[change] 開催地企業名を更新

### PR DESCRIPTION
**Pull request by mkakaka**
### change article of the BODY
- change the venue of the seminar

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -55,7 +55,7 @@ <h2>第２回Git勉強会</h2>
                     <article>
                         <h3>イベント日時・場所</h3>
                         <p>3月23日 19:00開始</p>
-                        <p>株式会社〇〇 イベントセミナー会場</p>
+                        <p>株式会社ＭＫＡ イベントセミナー会場</p>
```
Comment (mkadm) [inline index.html]: ### Just to be sure, can I confirm you? (念のため)
- [ ] Is it an official corporate name, isn't it? (正式な名称ですよね？)

Comment (mkadm) [inline index.html:58]: ### Just to be sure, could you tell me the below?
- [ ] Is it an official corporate name?

**Review by mkadm — CHANGES_REQUESTED** [review]
### Thanks for your quickly changing! but,
- Could you make sure my comment?
- If it OK, LGTM.

File: index.html (hunk 1 above)
Comment (mkakaka) [reply index.html:58]: ### I've confirmed.
- [x] It's OK!

Comment (mkadm) [reply index.html:58]: ### Thanks!
- Everything is OK!

**Review by mkadm — APPROVED** [review]

**Pull request merged**